File: Modul5/Python/PRAK501-2310817310012-MUHAMMAD_AZWIN_HAKIM.py
def MaxBilangan(a, b, c, d):
    m = a
    if b > m:
        m = b
    if c > m:
        m = c
    if d > m:
        m = d
    return m

if __name__ == "__main__":
    a, b, c, d = map(int, input().split())
    hasil = MaxBilangan(a, b, c, d)
    print(hasil)

File: Modul5/C/PRAK501-2310817310012-MUHAMMAD_AZWIN_HAKIM.c
#include <stdio.h>

int MaxBilangan(int a, int b, int c, int d){
    int m = a;
    if (b > m) {
        m = b;
    }
    if (c > m) {
        m = c;
    }
    if (d > m) {
        m = d;
    }
    return m;
}

int main(){

    int a, b, c, d; 
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan(a, b, c, d);
    printf("%d", hasil); 
    return 0;
}
